rules/windows/builtin/win_mal_service_installs.yml

```diff
@@ -1,31 +0,0 @@
-title: Malicious Service Installations
-id: 5a105d34-05fc-401e-8553-272b45c1522d
-description: Detects known malicious service installs that only appear in cases of lateral movement, credential dumping and other suspicious activity
-author: Florian Roth, Daniil Yugoslavskiy, oscd.community (update)
-date: 2017/03/27
-modified: 2019/11/01
-tags:
-    - attack.persistence
-    - attack.privilege_escalation
-    - attack.t1050
-    - car.2013-09-005
-logsource:
-    product: windows
-    service: system
-detection:
-    selection:
-        EventID: 7045
-    malsvc_wce:
-        ServiceName:
-            - 'WCESERVICE'
-            - 'WCE SERVICE'
-    malsvc_paexec:
-        ServiceFileName|contains: '\PAExec'
-    malsvc_wannacry:
-        ServiceName: 'mssecsvc2.0'
-    malsvc_persistence:
-        ServiceFileName|contains: 'net user'
-    condition: selection and 1 of malsvc_*
-falsepositives:
-    - Penetration testing
-level: critical
```

Comment:
I wouldn't delete this rule, but only lines 18-21:

https://github.com/Neo23x0/sigma/blob/77c927bc140d0643ca524e7f2755d7700eab5204/rules/windows/builtin/win_mal_service_installs.yml#L18-L21

as that part is covered in:

https://github.com/Neo23x0/sigma/blob/f11883966457e969a5f1f54f75814bd717cf59a4/rules/windows/builtin/win_mal_creddumper.yml#L38-L42

The rest of stuff is not relevant for the `rules/windows/builtin/win_mal_creddumper.yml`, as it is dedicated for **Credential dumping tools service execution**. I will provide more details commenting that rule, please give me a few minutes.